numpy/random/bit_generator.pyi

```diff
@@ -0,0 +1,88 @@
+import abc
+from threading import Lock
+from typing import Literal, NamedTuple, Optional
```

Comment:
Could you run mypy over `bit_generator.pyi`? It appears that a number of `typing` imports are still missing, _e.g._ `Any`, `List` and `Dict`.

Reply:
Thanks.  I was missing a lot. Too much reliance on PyCharm.  Fixed now.  

There are 4 errors:

```
numpy/random/bit_generator.pyi:55: error: Return type "List[SeedSequence]" of "spawn" incompatible with return type "List[ISpawnableSeedSequence]" in supertype "ISpawnableSeedSequence"
numpy/random/bit_generator.pyi:74: error: Overloaded function signatures 1 and 2 overlap with incompatible return types
numpy/random/bit_generator.pyi:74: error: Overloaded function signatures 1 and 3 overlap with incompatible return types
numpy/random/bit_generator.pyi:78: error: Overloaded function signatures 2 and 3 overlap with incompatible return types
```

I have looked at them but can't really understand why they are occurring

Reply:
https://github.com/numpy/numpy/pull/18395#discussion_r574673956 should fix the first error.

The other two are kinda sorta false positives related to the keyword arguments being a hassle.
You can either ignore them or mark the respective lines with a `# type: ignore[misc]` comment to silence the errors.